scripts/setup-smoke-config.sh

```diff
@@ -3,7 +3,6 @@ set -o errexit
 
 mkdir -p clients
 cat >> testdata/smoke_config.yml <<EOF
-log_path: "STDOUT"
```

Comment:
Why is this change necessary?

Reply:
Ah, because smoke_config.yml already has log_path defined (as server_logs.txt), so it was causing an error. https://github.com/evergreen-ci/evergreen/blob/105ce6cc43fe656bac0431cf479eb5d3b268567c/testdata/smoke_config.yml#L1 
However, I chose which one to drop kind of randomly. @bsamek do you know if one is preferable?